### PR TITLE
Upgrade rollup to the latest version

### LINKED ISSUE #1733: Update rollup dependency
We need to catch up to not get stuck on old versions, which kind of is the case at the moment.

Someone should update rollup and all its plugins and take a lot of time to tests if everything still works.

### PR DESCRIPTION
**Pull request by danyball**
Fixes #1733

# Done is when (DoD):
- [ ] Modifications within components are reflected by tests.
- [x] A self-review of the code has been performed.
- [ ] The modified component properties have been added to the typescript typings.
- [ ] Potential design changes have been reviewed by a designer.
- [ ] The modifications are well documented (README.md, etc.) and showcased in the stories.
- [x] The modifications are documented in the code, particularly in hard-to-understand areas (focus on **WHY**).
- [ ] The modifications are shortly added to CHANGELOG.md with the issue number.
- [x] The modifications still work in IE.
- [x] The modifications are according to our [Best Practices](https://github.com/axa-ch/patterns-library/blob/develop/CONTRIBUTION.md#best-practices)

### CODE REVIEW

**Comment by danyball** [comment]
Webpack build is running (checkout feature branch).

Rollup build is working. Check or compare the created files at /lib folders.

File: scripts/build/dist.rollup.js [hunk 1]
```diff
@@ -64,7 +64,7 @@ const dist = {
     }),
     babel({
       babelrc: false,
-      runtimeHelpers: true,
+      babelHelpers: 'runtime',
```
Comment (LucaMele) [inline scripts/build/dist.rollup.js:67]: nice

**Review by LucaMele — APPROVED** [review]
awesome. Did you check the build artefact? the `lib/index.js` and the `dist/index.js`? Important is that node modules are referenced and not bundled

**Comment by danyball** [comment]
> awesome. Did you check the build artefact? the `lib/index.js` and the `dist/index.js`? Important is that node modules are referenced and not bundled

Yes i did. `lib` is completly identical. `dist` has just one minimal change. No modules bundled.

**Pull request merged**